### ISSUE TITLE
Using "View subtree text" on solutions or projects causes error.

### ISSUE THREAD

**Issue by JohnLudlow** [KirillOsenkov/MSBuildStructuredLog#244]
Right-clicking a project or solution and choosing "View subtree text" causes an error. On most other items (e.g targets or tasks) you get some log text.

# Steps to Reproduce
1. Do a build with a binary log
2. Open the binary log in the log viewer
3. Find a project or solution node in the log, right click and choose "View subtree text"

# Expected
A log pane with text, similar to a target or task

# Actual
```
---------------------------

---------------------------
Unexpected exception. Sorry about that.

Please Ctrl+C to copy this text and file an issue at https://github.com/KirillOsenkov/MSBuildStructuredLog/issues/new



System.ArgumentException: Illegal characters in path.

   at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)

   at System.IO.Path.GetFileName(String path)

   at StructuredLogViewer.Controls.SourceFileTabHeader..ctor(SourceFileTab sourceFileTab)

   at StructuredLogViewer.Controls.DocumentWell.DisplaySource(String sourceFilePath, String text, Int32 lineNumber, Int32 column, Action preprocess, Boolean displayPath)

   at StructuredLogViewer.Controls.BuildControl.DisplayText(String text, String caption)

   at StructuredLogViewer.Controls.BuildControl.ViewSubtreeText()

   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)

   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)

   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)

   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
---------------------------
OK   
---------------------------
```

# Notes
* I actually saw this on a few other node types but this seemed to be inconsistent.
* If for some reason it's not possible to show the log text for a node then the option to do that shouldn't be available.

**Comment by KirillOsenkov** [comment]
Thanks for reporting! Fixed in 2.0.91. To update, just start the viewer from Windows start menu, wait for it to auto-update to 2.0.91, then restart again.

**Comment by JohnLudlow** [comment]
Thanks for the quick fix!
